Packs/MapValuesTransformer/Scripts/MapValuesTransformer/MapValuesTransformer.py

```diff
@@ -0,0 +1,58 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import json
+
+
+def mapValues(value, input_values, mapped_values):
```

Comment:
add unit test. No need for test playbook.
create new file `MapValuesTransformer_test.py`
https://xsoar.pan.dev/docs/integrations/unit-testing

Reply:
WIll attempt this ;o)